sphinxcontrib_hdl_diagrams/__init__.py

```diff
@@ -237,18 +238,20 @@ def run(self):
         return [node]
 
 
-def run_yosys(src, cmd, yosys='yowasp'):
+def run_yosys(src, cmd, yosys='yowasp', opt=''):
```

Comment:
I'm assuming that `opt` is short for `options` and not `optimization`? I suggest we use the longer name to prevent confusion.

Reply:
That's a good idea.